service/.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules
 build
 .env
 tsconfig.tsbuildinfo
-peer-id.json
\ No newline at end of file
+peer-id.json
+typechain
```

Comment:
Although the typechain docs say to not include typechain in git since it is a generated folder, I would suggest leaving it here since there is no way to generate it from within the arch service project. It's basically just a static types folder. You could even name it `types` or something so it's not confused with a generated typechain folder.

Reply:
Ah, good catch. Yep